packages/cli/src/commands/join.ts

```diff
@@ -58,6 +59,27 @@ packageVersion: string
     entrypoints.map(ref => externalRefResolver.resolveDocument(null, ref, true) as Promise<Document>)
   );
 
+  const bundleResults = await Promise.all(
```

Comment:
@ohorbachevskyi Hi! Is it possible that introducing a bundle inside the join command now runs all the decorators/preprocessor on my files before joining them? I noticed this regression after bump my version to the latest